File: GTTS/INL.UserService.Models/UserProfile_Item.cs
using System.Collections.Generic;

namespace INL.UserService.Models
{
	public class UserProfile_Item : IUserProfile_Item
	{
		public int AppUserID { get; set; }
		public string ADOID { get; set; }
		public string First { get; set; }
		public string Middle { get; set; }
		public string Last { get; set; }
		public string FullName { get; set; }
		public string PositionTitle { get; set; }
		public string EmailAddress { get; set; }
		public string PhoneNumber { get; set; }
		public string PicturePath { get; set; }
		public int? CountryID { get; set; }
		public string CountryName { get; set; }
		public int? PostID { get; set; }
		public string PostName { get; set; }
		public IBusinessUnit_Item DefaultBusinessUnit { get; set; }
		public IAppRole_Item DefaultAppRole { get; set; }
		public int ModifiedByAppUserID { get; set; }

		public List<IAppRole_Item> AppRoles { get; set; }
		public List<IAppPermission_Item> AppPermissions { get; set; }
		public List<IBusinessUnit_Item> BusinessUnits { get; set; }
	}
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UserService.Models/user-profile_item.ts


import { IBusinessUnit_Item } from './ibusiness-unit_item';
import { IAppRole_Item } from './iapp-role_item';
import { IAppPermission_Item } from './iapp-permission_item';
import { IUserProfile_Item } from './iuser-profile_item';

export class UserProfile_Item implements IUserProfile_Item {
  
	public AppUserID: number = 0;
	public ADOID: string = "";
	public First: string = "";
	public Middle: string = "";
	public Last: string = "";
	public FullName: string = "";
	public PositionTitle: string = "";
	public EmailAddress: string = "";
	public PhoneNumber: string = "";
	public PicturePath: string = "";
	public CountryID?: number;
	public CountryName: string = "";
	public PostID?: number;
	public PostName: string = "";
	public DefaultBusinessUnit?: IBusinessUnit_Item;
	public DefaultAppRole?: IAppRole_Item;
	public ModifiedByAppUserID: number = 0;
	public AppRoles?: IAppRole_Item[];
	public AppPermissions?: IAppPermission_Item[];
	public BusinessUnits?: IBusinessUnit_Item[];
  
}


